Empire/Empire.cpp

```diff
@@ -1458,6 +1498,8 @@ int Empire::StockpileID(ResourceType res) const {
         return m_capital_id;
         break;
     case RE_INDUSTRY:
+        return IMPERIAL_OBJECT_ID;
```

Comment:
just get rid of the concept of a stockpile location, rather than adding some fake ID to use for it...